Client/Frontend/Widgets/SimpleHighlightCell.swift

```diff
@@ -0,0 +1,191 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import Shared
+import Storage
+
+struct SimpleHighlightCellUX {
+    static let LabelColor = UIAccessibilityDarkerSystemColorsEnabled() ? UIColor.blackColor() : UIColor(rgb: 0x353535)
+    static let BorderWidth: CGFloat = CGFloat(0.5)
+    static let CellSideOffset = 20
+    static let AlternateBottomOffset = 16
+    static let TitleLabelOffset = 10
+    static let CellTopBottomOffset = 12
+    static let SiteImageViewSize = 48
+    static let StatusIconSize = 12
+    static let DescriptionLabelColor = UIColor(colorString: "919191")
+    static let TimestampColor = UIColor(colorString: "D4D4D4")
+    static let SelectedOverlayColor = UIColor(white: 0.0, alpha: 0.25)
+    static let PlaceholderImage = UIImage(named: "defaultTopSiteIcon")
+    static let CornerRadius: CGFloat = 3
+    static let NearestNeighbordScalingThreshold: CGFloat = 24
+}
+
+class SimpleHighlightCell: UITableViewCell {
+    var siteImage: UIImage? = nil {
+        didSet {
+            if let image = siteImage {
+                siteImageView.image = image
+                siteImageView.contentMode = UIViewContentMode.ScaleAspectFit
+
+                // Force nearest neighbor scaling for small favicons
+                if image.size.width < SimpleHighlightCellUX.NearestNeighbordScalingThreshold {
+                    siteImageView.layer.shouldRasterize = true
+                    siteImageView.layer.rasterizationScale = 2
+                    siteImageView.layer.minificationFilter = kCAFilterNearest
+                    siteImageView.layer.magnificationFilter = kCAFilterNearest
+                }
+            } else {
+                siteImageView.image = SimpleHighlightCellUX.PlaceholderImage
+                siteImageView.contentMode = UIViewContentMode.Center
+            }
+        }
+    }
+
+    lazy var titleLabel: UILabel = {
+        let titleLabel = UILabel()
+        titleLabel.font = DynamicFontHelper.defaultHelper.DeviceFontMediumBold
+        titleLabel.textColor = SimpleHighlightCellUX.LabelColor
+        titleLabel.textAlignment = .Left
+        titleLabel.numberOfLines = 3
+        return titleLabel
+    }()
+
+    lazy var descriptionLabel: UILabel = {
+        let titleLabel = UILabel()
+        titleLabel.setContentHuggingPriority(1000, forAxis: UILayoutConstraintAxis.Vertical)
+        titleLabel.font = DynamicFontHelper.defaultHelper.DeviceFontDescriptionActivityStream
+        titleLabel.textColor = SimpleHighlightCellUX.DescriptionLabelColor
+        titleLabel.textAlignment = .Left
+        titleLabel.numberOfLines = 1
```

Comment:
I'm assuming it just truncates the text - but they're just 1 - 2 words (e.g. Bookmarked, Open Tab, Visited, Reading List) so hopefully nothing would get chopped off(?) Would ping Bryan Bell for more info if that were to happen